xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -2862,6 +2862,10 @@ bool CLinuxRendererGL::Supports(ESCALINGMETHOD method)
     if (scaleX < minScale && scaleY < minScale)
       return false;
 
+    // avoid using hq scalers when downscaling
```

Comment:
The question is rather: what do you gain?

The input is let's say 3840x2160 and you want to display it on a 720x360 window. You really think that you will see the difference between lanczos3 optimized and bilinear downscaling?